### ISSUE TITLE
21344518: Xcode-beta (7A120f): Swift compiler crashes when I build my app

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#990]
#### Description

Assertion failed: (hasAccessibility() && "accessibility not computed yet"), function getFormalAccess, file /Library/Caches/com.apple.xbs/Sources/swiftlang/swiftlang-700.0.38.1/src/swift/include/swift/AST/Decl.h, line 2189.

Full stack trace here:
https://gist.github.com/davbeck/5680b26813a821147bf9

It seems to happen in any Swift function that overrides another Swift function (cellForRowAtIndexPath: in this case).
## 

Product Version: Xcode-beta (7A120f)
Created: 2015-06-11T19:43:00.191120
Originated: 2015-06-11T12:42:00
Open Radar Link: http://www.openradar.me/21344518
